src/backends/interflop-vprec/interflop_vprec.c

```diff
@@ -273,34 +378,87 @@ void _set_vprec_inst_mode(vprec_inst_mode mode) {
 // Round the float with the given precision
 static float _vprec_round_binary32(float a, char is_input, void *context,
                                    int binary32_range, int binary32_precision) {
+  t_context *currentContext = (t_context *)context;
+
+  /* test if 'a' is a special case */
   if (!isfinite(a)) {
     return a;
   }
 
-  /* round to zero or set to infinity if underflow or overflow compare to
+  /* round to zero or set to infinity if underflow or overflow compared to
    * VPRECLIB_BINARY32_RANGE */
   int emax = (1 << (binary32_range - 1)) - 1;
-  // here emin is the smallest exponent in the *normal* range
+  /* here emin is the smallest exponent in the *normal* range */
   int emin = 1 - emax;
 
+  // /* in absolute error mode, the error threshold also gives the possible
```

Comment:
Why do we keep this commented block ?

Reply:
There's no need to keep this block.
It was an oversight.
Now removed, from both _vprec_round_binary32 and _vprec_round_binary64.